### PR TITLE
Account for a system-wide installed bundler package

### LINKED ISSUE #217: bundler gem not found on bootstrap
On new install got the message "gem install bundler returned successfully, but still cannot find bundler". Originating from here:
https://github.com/rock-core/autoproj/blob/4ae9a7c3fb908da7bdf2639d0c3915119a60ab97/bin/autoproj_bootstrap#L295

Investigation shows gem which returns system's bundler
```
$ GEM_HOME="/home/vbargsten/.autoproj/gems/ruby/2.5.0" GEM_PATH="/home/vbargsten/.autoproj/gems/ruby/2.5.0" gem which bundler
/usr/lib/ruby/vendor_ruby/bundler.rb
```

Calling with `-a` to give all results:

```
$ GEM_HOME="/home/vbargsten/.autoproj/gems/ruby/2.5.0" GEM_PATH="/home/vbargsten/.autoproj/gems/ruby/2.5.0" gem which -a bundler
/usr/lib/ruby/vendor_ruby/bundler.rb                                                                                                                                                         
/home/vbargsten/.autoproj/gems/ruby/2.5.0/gems/bundler-1.16.4/lib/bundler.rb
```

A quick fix for me was to call with `-g`:
```
$ GEM_HOME="/home/vbargsten/.autoproj/gems/ruby/2.5.0" GEM_PATH="/home/vbargsten/.autoproj/gems/ruby/2.5.0" gem which -g bundler                            
/home/vbargsten/.autoproj/gems/ruby/2.5.0/gems/bundler-1.16.4/lib/bundler.rb
```

System is Ubuntu 18.04

### PR DESCRIPTION
**Pull request by 2maz**
To fix #217

### CODE REVIEW

File: lib/autoproj/ops/install.rb [hunk 1]
```diff
@@ -281,15 +281,16 @@ def parse_options(args = ARGV)
...
+                setup_paths =
+                    IO.popen([env_for_child, Gem.ruby, gem_program, 'which','-a', 'bundler/setup']) do |io|
                         io.read
                     end
                 return unless $?.success?
-
                 bundler_path = File.join(gems_gem_home, 'bin', 'bundle')
-                if File.exist?(bundler_path) && setup_path.start_with?(gems_gem_home)
-                    bundler_path
+                setup_paths.split("\n").each do |setup_path|
```
[the remark this answers is not in the document]
Comment (2maz) [reply lib/autoproj/ops/install.rb]: Agreed. Updated accordingly.

**Comment by 2maz** [comment]
@doudou @g-arjones any further comments?

**Comment by 2maz** [comment]
K, but does a merge onto master requires releasing the gem?!
IMO having the fixed bootstrap script available through the website will already catch (most of) the open related issues (FYI: I already use the script for testing on https://github.com/rock-core/rock-osdeps-package_set/blob/master/Dockerfile +26)

**Comment by g-arjones** [comment]
Nope.. Ok, then!

**Pull request merged**